pkg/api/unversioned/generated.proto

```diff
@@ -21,6 +21,7 @@ syntax = 'proto2';
 
 package k8s.io.kubernetes.pkg.api.unversioned;
 
+import "k8s.io/kubernetes/pkg/runtime/generated.proto";
```

Comment:
Not sure - could be an input dependency order.  I'll open an issue to have
proto only generate imports it uses.

On Thu, May 5, 2016 at 10:00 AM, Wojciech Tyczynski <
notifications@github.com> wrote:

> In pkg/api/unversioned/generated.proto
> <https://github.com/kubernetes/kubernetes/pull/24710#discussion_r62190391>
> :
>
> > @@ -21,6 +21,7 @@ syntax = 'proto2';
> >
> >  package k8s.io.kubernetes.pkg.api.unversioned;
> >
> > +import "k8s.io/kubernetes/pkg/runtime/generated.proto";
>
> Hmm - I don't understand.
> The rest of the file didn't change - so why did the imports change?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24710/files/e416be61b90b5dc461dc8a84a94d6ecd0d76c216#r62190391>
>